synchronize-with-npm/entrypoint.sh

```diff
@@ -26,8 +26,8 @@ if [ "${#NPM_AUTH_TOKEN}" -eq "0" ]
         git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
         git config user.name "$GITHUB_ACTOR"
         
-        git tag -f "v$version"
-        git push "https://$GITHUB_ACTOR:$GITHUB_TOKEN@github.com/$GITHUB_REPOSITORY.git" --tag
+        git tag --force "v$version"
+        git push --force "https://$GITHUB_ACTOR:$GITHUB_TOKEN@github.com/$GITHUB_REPOSITORY.git" --tag
```

Comment:
Can we move the `--tag` option to the front too?

Reply:
done